Examples/frame-sheet-name-test.lisp

```diff
@@ -0,0 +1,38 @@
+(cl:in-package #:clim-demo)
+
+(define-application-frame frame-sheet-name-test ()
+  ()
+  (:pane
+   (let* ((frame *application-frame*)
+          (sheet (frame-top-level-sheet frame)))
+     (tabling ()
```

Comment:
Frame looks pretty crowded, Try something like below to add some whitespace around elements (Feel free to tweak values if you see fit):
```lisp
(define-application-frame frame-sheet-name-test ()
  ()
  (:menu-bar nil)
  (:pane
   (let* ((frame *application-frame*)
          (sheet (frame-top-level-sheet frame)))
     (outlining (:thickness 10 :background +gray84+)
       (tabling (:min-width 600 :spacing 8)
	 (list (labelling (:label  " "))
	       (labelling (:label "Name"))
	       (labelling (:label "Pretty Name")))
	 (list (labelling (:label "Sheet"))
	       (make-pane :label :label (prin1-to-string (sheet-name sheet)))
	       (outlining (:thickness 4 :background +white+)
		 (make-pane :text-field :name 'sheet-pretty-name
					:value (clime:sheet-pretty-name sheet)
					:value-changed-callback
					(lambda (gadget new-value)
					  (declare (ignore gadget))
					  (setf (clime:sheet-pretty-name sheet)
						new-value)))))
	 (list (labelling (:label "Frame"))
	       (make-pane :label :label (prin1-to-string (frame-name frame)))
	       (outlining (:thickness 4 :background +white+)
		 (make-pane :text-field :value (frame-pretty-name frame)
					:value-changed-callback
					(lambda (gadget new-value)
					  (declare (ignore gadget))
					  (setf (frame-pretty-name frame)
						new-value)))))))))
  (:default-initargs
   :pretty-name "Frame and Sheet Name Test"))
```